storage/rocksdb/ha_rocksdb.cc

```diff
@@ -6327,17 +6343,23 @@ int ha_rocksdb::read_row_from_secondary_key(uchar *const buf,
   int rc = 0;
   uint pk_size;
 
-  if (m_keyread_only && m_lock_rows == RDB_LOCK_NONE && !has_hidden_pk(table)) {
-    /* Get the key columns and primary key value */
-    const rocksdb::Slice &rkey = m_scan_it->key();
+  /* Get the key columns and primary key value */
+  const rocksdb::Slice &rkey = m_scan_it->key();
+  const rocksdb::Slice &value = m_scan_it->value();
+
+  DBUG_ASSERT_IMP(!m_keyread_only, !m_lookup_bitmap_vect.empty());
```

Comment:
It seems to me like m_lookup_bitmap_vect is never empty since we always fill it up in setup_lookup_bitmaps? Maybe you meant the converse instead?